e2e/dashboards-suite/Repeating_a_panel_horizontally.spec.ts

```diff
@@ -2,14 +2,49 @@ import { e2e } from '@grafana/e2e';
 const PAGE_UNDER_TEST = 'WVpf2jp7z/repeating-a-panel-horizontally';
 
 describe('Repeating a panel horizontally', () => {
-  it('should be able to repeat a panel horizontally', () => {
+  beforeEach(() => {
     e2e.flows.login('admin', 'admin');
+  });
+
+  it('should be able to repeat a panel horizontally', () => {
     e2e.flows.openDashboard({ uid: PAGE_UNDER_TEST });
+    let prevLeft = Number.NEGATIVE_INFINITY;
+    let prevTop = null;
+    const panelTitles = ['Panel Title 1', 'Panel Title 2', 'Panel Title 3'];
+    panelTitles.forEach((title) => {
+      e2e.components.Panels.Panel.title(title)
+        .should('be.visible')
+        .then(($el) => {
+          const { left, top } = $el[0].getBoundingClientRect();
+          expect(left).to.be.greaterThan(prevLeft);
+          if (prevTop !== null) {
+            expect(top).to.be.equal(prevTop);
+          }
+
+          prevLeft = left;
+          prevTop = top;
+        });
+    });
+  });
 
+  it('responds to changes to the variables', () => {
+    e2e.flows.openDashboard({ uid: PAGE_UNDER_TEST });
     let prevLeft = Number.NEGATIVE_INFINITY;
     let prevTop = null;
     const panelTitles = ['Panel Title 1', 'Panel Title 2', 'Panel Title 3'];
     panelTitles.forEach((title) => {
+      e2e.components.Panels.Panel.title(title).should('be.visible');
+    });
+
+    // Change to only show panels 1 + 3
+    e2e.pages.Dashboard.SubMenu.submenuItemLabels('horizontal').click();
+    e2e.pages.Dashboard.SubMenu.submenuItemValueDropDownOptionTexts('1').click();
+    e2e.pages.Dashboard.SubMenu.submenuItemValueDropDownOptionTexts('3').click();
+    e2e.pages.Dashboard.DashNav.navV2().click();
```

Comment:
this is a hack which works but isn't particularly robust... would love a better suggestion for this bit. we need to blur the dropdown menu for the changes to be processed, but i can't find a way to blur it/anything suitable to select to force a blur. in the UI, you would click "outside" of the dropdown.

Reply:
Maybe something like 
```suggestion
    e2e().get('body').click();
```

?

Reply:
yep, that's much nicer 😄  thanks!